src/Space/Create/CreateSpaceAction.php

```diff
@@ -18,7 +16,7 @@
 use Symfony\Component\Routing\RouterInterface;
 use Twig\Environment as Twig;
 
-final class Create
+final class CreateSpaceAction
```

Comment:
your `Action` classes should implement `App\Action` so that dependency injection works correctly (see `services.yaml`)